mhs/operating-mhs-adaptor.md

```diff
@@ -91,30 +59,215 @@ An example is given below:
+| 7 | `Looking up endpoint details for service_id=urn:nhs:names:services:pdsquery:QUPA_IN040000UK32` | Request made to routing details from spine route lookup |
+| 8 | `Retrieved endpoint details for details="{'service_id': 'urn:nhs:names:services:pdsquery:QUPA_IN040000UK32', 'url': 'https://192.168.128.11/sync-service', 'party_key': 'YES-0000806', 'cpa_id': 'S20001A000168', 'to_asid': '928942012545'}"` | Confirmation endpoint successfully retrieved routing details from spine route lookup | 
+| 9 | `Attempting to publish work description key=C78164D9-A1EB-497E-BCB5-74CABD3162F0` | Attempts to publish the local state of the work description to the state store to update message state from this information received spine |
+| 10 | `Retrieving latest work description to check version` | Queries state database with local state to check remote state exists |
+| 11 | `Retrieved previous version, comparing versions` | Previous remote version found in state database and comparing versions |
+| 12 | `Local version matches remote, incrementing local version number` | Checks versions to avoid collisions. Previous remote version matches local version, increments local version number | 
+| 13 | `Successfully updated work description to state store for key=C78164D9-A1EB-497E-BCB5-74CABD3162F0` | Successfully updated state database with message returned from spine route lookup | 
+| 14 | `About to make outbound request` | Starting process to send message request from Outbound to Spine |
+| 15 | `Outbound Synchronous workflow completed. Message sent to Spine and Acknowledgment=MessageStatus.OUTBOUND_MESSAGE_RESPONSE_RECEIVED received.` | Message successfully sent to spine from outbound and spine returned acknowledgement | 
+| 16 | `Attempting to publish work description key=C78164D9-A1EB-497E-BCB5-74CABD3162F0` | Attempts to publish the local state of the work description to the state store to update message state from this information received spine |
+| 17 | `Retrieving latest work description to check version` | Queries state database with local state to check remote state exists |
+| 18 | `Retrieved previous version, comparing versions` | Previous remote version found in state database and comparing versions |
+| 19 | `Local version matches remote, incrementing local version number` | Checks versions to avoid collisions. Previous remote version matches local version, increments local version number | 
+| 20 | `Successfully updated work description to state store for key=C78164D9-A1EB-497E-BCB5-74CABD3162F0` | Successfully updated state database with message status updated, spine successfully received message. | 
+
+### Cloud Watch
+
+See below example to write CloudWatch queries to parse logs and search for a given correlation id and filter out healthchecks. 
+
+ `parse '* | * | * | * | * | * | * | * | *' as timestamp, level, correlation_id, message_id, interaction_id, inbound_message_id, process, name, text `<br>
```

Comment:
Rested this query after https://github.com/nhsconnect/integration-adaptors/pull/200/files was merged and it correctly captures outbound logs. I didn't see any for inbound though.